src/test/groovy/com/mateusz/jakuszko/roomforyou/mapper/ApartmentMapperSpecification.groovy

```diff
@@ -0,0 +1,118 @@
+package com.mateusz.jakuszko.roomforyou.mapper
+
+import com.mateusz.jakuszko.roomforyou.dto.ApartmentDto
+import com.mateusz.jakuszko.roomforyou.entity.Apartment
+import com.mateusz.jakuszko.roomforyou.entity.Customer
+import com.mateusz.jakuszko.roomforyou.entity.Reservation
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import spock.lang.Specification
+
+@SpringBootTest
+class ApartmentMapperSpecification extends Specification {
+
+    @Autowired
+    private ApartmentMapper apartmentMapper
+
+    private static Apartment createApartment() {
```

Comment:
W ramach dobrych praktyk proponuje w klasach testowych zawsze taką kolejność:

1. pola
2. metoda setup() (albo odpowiednik zależnie od środowiska)
3. testy
4. prywatne metody pomocnicze

Dzięki temu łatwiej o czytelność - to co interesuje nas najbardziej czyli same testy znajdują się (prawie) na szczycie pliku